src/helpers.js

```diff
@@ -56,3 +56,11 @@ export function idsToIgnoredRules(ruleIds = []) {
     return ids
   }, {})
 }
+
+export function validatePoint(textEditor, line, col) {
+  const buffer = textEditor.getBuffer()
+  // Clip the given point to a valid one, and check if it equals the original
+  if (!buffer.clipPosition([line, col]).isEqual([line, col])) {
+    throw new Error(`${line}:${col} isn't a valid point!`)
```

Comment:
This should be validating the point in a similar manner to how `rangeFromLineNumber()` does. Where this is being called [over here](https://github.com/AtomLinter/linter-eslint/pull/709/files/86381852cd4f81dc140230e4270a7347e9974ba1#diff-1c90ff38a08209f9ebd4d05d1e43358eR173) is wrapped in the same `try`/`catch` that the call to `rangFromLineNumber()`, so this `throw` will be caught and handled in the exact same manner as any other invalid point we run into.